src/software/util/typename/typename.h

```diff
@@ -24,4 +40,4 @@ std::string demangle_typeid(const char* mangled_name);
  *
  * @return the string representation of the object
  */
-#define TYPENAME(object) (demangle_typeid(typeid(object).name()))
+#define TYPENAME(object) (demangleTypeId(typeid(object).name()))
```

Comment:
do we still need this if we can use `objectTypeName` for everything?

Reply:
we can't use objectTypeName for classes, e.g.  `class MyClass; objectTypeName(MyClass)` doesn't work because MyClass is not an object reference